### PR TITLE
Add multi-answer text support

### PR DESCRIPTION
**Pull request by BarnabeD**
First part : the easy part, text splitting.

I'm not sure about naming `MultiPrompt`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,20 @@ name = Prompts::TextPrompt.ask(
 
 The block will receive the value that has been entered and may return an error message, or `nil` if the validation passes.
 
+#### Multiple answer
+
+If you would like recieve multiple responses from a single prompt, you can use `MultiPrompt`. The defauls sep
```
Comment (fractaledmind) [inline README.md:110]:
```suggestion
If you would like recieve multiple responses from a single prompt, you can use `MultiSelectPrompt`.
```

File: lib/prompts.rb [hunk 2]
```diff
@@ -18,6 +18,7 @@
 require_relative "prompts/text_prompt"
 require_relative "prompts/select_prompt"
 require_relative "prompts/form"
+require_relative "prompts/multi_prompt"
```
Comment (fractaledmind) [inline lib/prompts.rb:21]:
```suggestion
require_relative "prompts/multiselect_prompt"
```

File: lib/prompts/multi_prompt.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Prompts
+  class MultiPrompt < Prompt
```
Comment (fractaledmind) [inline lib/prompts/multi_prompt.rb:4]:
```suggestion
  class MultiSelectPrompt < Prompt
```

File: lib/prompts/multi_prompt.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    def initialize(...)
+      super
+
+      @separator ||= ","
+      @instructions = "Press Enter to submit"
+      @hint ||= "Type your responses separated by #{humanized_separator} and press Enter ⏎"
```
Comment (fractaledmind) [inline lib/prompts/multi_prompt.rb:10]:
```suggestion
      @hint ||= "Type your responses separated by #{humanized_separator} (#{@separator}) and press Enter ⏎"
```

**Comment by BarnabeD** [comment]
@fractaledmind thanks for your feedback !

I wonder if `MultiSelectPrompt` isn't too similar to the `Select multiple:true` concept?
What do you think about `MultiTextPromp`?

**Comment by fractaledmind** [comment]
Back from vacation and looking at this again. With fresh eyes I can see that I don't think this belongs in the core library. The concept of `Prompts` is to provide CLI inputs that map to web inputs. These inputs are then building blocks upon which users can build whatever they need. There is no web input for multiple text values passed to a single text input. And this really only tweaks the instructions and then parses the result of a TextPrompt. This feels like something that if a user needs, they should simply implement themselves on top of the TextPrompt class. I would be open, instead, to a small section in the README demonstrating how to extend a base prompt class to provide a custom prompt.

**Comment by BarnabeD** [comment]
Yes, that's right, that's what I've done in my application, but it looks a bit flaky indeed.

**Pull request closed without merging**